### PR TITLE
Use JATS for APS

### LINKED ISSUE #184: Use more metadata from APS
## Expected Behavior

Currently, the APS spider is only using the metadata provided by the JSON API. Instead, it should use the full data present in the bagit format, which contains the fulltext in JATS format from which more metatadata can be extracted and also figures, that we might extract for Open Access papers.

### PR DESCRIPTION
**Pull request by szymonlopaciuk**
## Description
For APS open access papers use the JATS format to extract more information about the papers, such as references.

## Related Issue
closes #184 and #39

## Checklist:

- [x] I have all the information that I need (if not, move to `RFC` and look for it).
- [x] I linked the related issue(s) in the corresponding commit logs.
- [x] I wrote [good commit log messages](https://github.com/torvalds/subsurface-for-dirk/blob/5f15ad5a86ada3c5e574041a5f9d85235322dabb/README#L92-L119).
- [x] My code follows the code style of this project.
- [x] I've added any new docs if API/utils methods were added.
- [x] I have updated the existing documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: hepcrawl/spiders/aps_spider.py [hunk 1]
```diff
@@ -76,53 +77,24 @@ def start_requests(self):
...
-            record.add_value('collections', ['HEP', 'Citeable', 'Published'])
-
-            parsed_item = ParsedItem(
-                record=record.load_item(),
-                record_format='hepcrawl',
-            )
-
-            yield parsed_item
+            if doi:
+                request = Request(url='{}/{}'.format(self.aps_base_url, doi),
```
Comment (michamos) [inline hepcrawl/spiders/aps_spider.py]: you should also attach the `xml` file (as a `hidden` `fulltext`) to the record with `add_document`.

Comment (szymonlopaciuk) [reply hepcrawl/spiders/aps_spider.py]: How do I know the `key`? Do I have to attach the file myself using `invenio-records-files`, or do we already have a facility for this somewhere in Inspire?

Comment (michamos) [reply hepcrawl/spiders/aps_spider.py]: the key should be the file name

File: hepcrawl/parsers/jats.py [hunk 2]
```diff
@@ -507,53 +516,98 @@ def get_reference(self, ref_node):
...
+        Args:
+            collaboration (string): collaboration name
 
-        return builder.obj
+        Returns:
+            string: sanitized collaboration name
+        """
+        if collaboration.startswith('(') and collaboration.endswith(')'):
+            collaboration = collaboration[1:-1]
+        collaboration = re.sub(r'\sCollaboration', '', collaboration)
```
Comment (michamos) [inline hepcrawl/parsers/jats.py]: this is already handled by the builder. The parentheses handling seems more generally useful, and should be handled there too.

**Pull request merged**